airqo-platform/AirQo-api
# Data mgt and incentives

- What does this PR do?
This PR created a new endpoint for sending mobile money to the device host's phone number.
The PR also shares the app structure for the data management service with a main focus on the weather forecast data as the third party data going to be stored and accessed via an API.
- What JIRA task is related to this PR?
The link to the JIRA board task is [here](https://airqoteam.atlassian.net/browse/APA-8).
- How do I test out this PR?
Git clone this repo `cd AirQo-api/src/incetives-service` Then run `npm install` to get all the necessary dependencies Afterwards run `npm run test` to test out the end points using any REST client tool available to you.
- Which end points should are ready for testing?
partial url: /api/v1/incentives/momo. More details about how to run this endpoint can be in [AirQo API documentation](https://docs.google.com/spreadsheets/d/1-NGTDBtTcIi8Fg2UYSWjcK6geKrZdopz5kxRFsCEMPs/edit#gid=530501335)
